src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -259,6 +259,11 @@ public X509Certificate2Collection ClientCertificates
         {
             get
             {
+                if (_clientCertificateOption != ClientCertificateOption.Manual)
```

Comment:
Had offline discussion with Caesar.  I think its ok to change WinHttpHandler here so that we have consistent behavior in the handlers.